regression.go

```diff
@@ -283,6 +287,41 @@ func (r *Regression) String() string {
 	return str
 }
 
+//Plot regression and return file path where its saved
+func (r *Regression) Plot() string {
```

Comment:
This should:
- [ ] return the `err` instead of panicking.
- [ ] allow the user to specify where to save the file
- [ ] needs to be more configurable in general. e.g. only the first two vars are plotted. What if there are more?